src/omnibar/omnibar.ts

```diff
@@ -10,8 +10,14 @@ export class BBOmnibar {
     }
 
     return new Promise<any>((resolve: any, reject: any) => {
-      BBOmnibarScriptLoader.registerScript(
-        'https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.0/jquery.js'
+      let jqueryVersion = '';
+      if (window.hasOwnProperty('$')) {
```

Comment:
Since this library will be supporting a variety of environments we should probably be very conservative in how we retrieve the jQuery version.  Who knows if someone has an environment where `$` is defined but isn't actually mapped to jQuery, or it is mapped to some jQuery drop-in replacement plugin that doesn't have an `fn` property and so on.  Maybe do something like this:

```
const jq = (<any> window).jQuery;
const jqVersion = jq && jq.fn && jq.fn.jquery;
```